hello-world/src/worker.ts

```diff
@@ -13,10 +15,12 @@ async function run() {
   // In order to configure the server connection parameters and other global options,
   // use the Core.install() method to configure the Rust Core SDK singleton.
   const worker = await Worker.create({
-    workDir: __dirname,
-    taskQueue: 'tutorial'
+    workflowsPath: path.join(__dirname, 'workflows'),
```

Comment:
Why is this change necessary? I think we generally encourage people to use `workDir`...

Reply:
Planning on removing workDir, replacing with something like:

![image](https://user-images.githubusercontent.com/251288/134029530-3bd654ae-8255-40bf-9510-9bae40c6eba6.png)

Reply:
Cool, I missed that convo. Thanks for pointing that out :+1: